### DISCUSSION TITLE
Use <nav> rather than <aside> for the ToC

### DISCUSSION THREAD

**Discussion by ghost** [mdn/yari#4693] [General]
I just realised by looking at the aside [page ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside#examples) that the Table of contents on the left side is an aside element. 

As this Table of contents displays navigation links I think that it should be a nav. Not that it would make a huge change, but it makes more sense semantically and in a page that explains what the HTML elements are it'd be the thing to do. 

This is element is the same in several pages so I'm not sure how the mark-up could be changed to nav, could anyone tell me how to make this happen?

Any help would be really appreciated :) 

Thanks a lot!

**Comment by wbamberg** [comment]
Thanks for this suggestion! As it's more about the platform than about content, I've transferred it to the mdn/yari repo.
